### PR TITLE
fix: modifies sidebar state such that it remains open

### PR DESCRIPTION
**Pull request by ayesha-waris**
[INF-915](https://2u-internal.atlassian.net/browse/INF-915)

- Removed state tracking and such that the sidebar always opens when a user navigates to a unit.
- The close button should still allow users to close the sidebar, but it should re-open when user reloads the page or navigates to another unit that has discussions enabled.

### CODE REVIEW

File: src/courseware/course/sidebar/SidebarContextProvider.jsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ const SidebarProvider = ({
       setCurrentSidebar(initialSidebar);
```
Comment (sundasnoreen12) [inline src/courseware/course/sidebar/SidebarContextProvider.jsx]: No need of line 26 const showDiscussionSidebar = localStorage.getItem('showDiscussionSidebar') !== 'false';

File: src/courseware/course/sidebar/SidebarContextProvider.jsx [hunk 2]
```diff
@@ -40,7 +40,7 @@ const SidebarProvider = ({
...
     }
```
Comment (sundasnoreen12) [inline src/courseware/course/sidebar/SidebarContextProvider.jsx:32]: for line 30-32 use this
const initialSidebar = SIDEBARS.DISCUSSIONS.ID.

File: src/courseware/course/sidebar/SidebarContextProvider.jsx (hunk 1 above)
Comment (sundasnoreen12) [inline src/courseware/course/sidebar/SidebarContextProvider.jsx]: remove check for line 23

**Review by sundasnoreen12 — CHANGES_REQUESTED** [review]

**Review by sundasnoreen12 — APPROVED** [review]
PR LGTM

**Review by awais-ansari — APPROVED** [review]

**Review by muhammadadeeltajamul — APPROVED** [review]

**Pull request merged**